macros/utils/table_operations/replace_table_data.sql

```diff
@@ -9,9 +9,12 @@
     {% do adapter.drop_relation(intermediate_relation) %}
 {% endmacro %}
 
+
 {# Spark / Databricks - truncate and insert (non-atomic) #}
 {% macro spark__replace_table_data(relation, rows) %}
-    {% do dbt.truncate_relation(relation) %}
```

Comment:
There's no `truncate_relation` in `dbt-spark`

Reply:
Interesting! 
Now I realize that dbt-databricks is using dbt-spark by default 
https://github.com/databricks/dbt-databricks/blob/25b2d801f62f70201ca4e86cd3e2629ba9b90244/tests/unit/macros/base.py#L47
Will do :)